proto_v2/slot_v2.proto

```diff
@@ -0,0 +1,68 @@
+syntax = "proto3";
+package fd.v2;
+
+import "txn_v2.proto";
+
+message SlotBank {
+    int32 todo = 1;
```

Comment:
flatten this into the slot env

Reply:
- recent_block_hashes
- prev_slot
- block_height
- last_restart_slot
- lamports_per_signature
- block_hash_queue